### PR TITLE
prettier logs when running strap

### PR DESCRIPTION
**Pull request by heathsnow**
@jtrinklein
@greglboxer 

Reduce the log output and add some pretty bash coloring.  Let me know if this is too much but I figure there's always the debug option for really digging into things.  This, imo, is a prettier output format to look at.

```bash
hsnow@localhost:~/Downloads $ ./strap.sh 
Enter your password (for sudo access):
Password:
--> Configuring security settings... ✔
--> Checking full-disk encryption status... ✔
--> Install Xcode Command Line Tools... ✔
--> Configuring Git... ✔
--> Setting up GitHub HTTPS credentials... ✔
--> Setting up GitHub SSH key... ✔
--> Check for Homebrew... ✔
--> Updating Homebrew... ✔
--> Installing Homebrew taps and extensions... ✔
--> Checking for software updates... ✔
--> Ensure strap is cloned locally and up to date... ✔
--> Checking for remotes on git@github.com:heathsnow/dotfiles... ✔
--> Fetching heathsnow/dotfiles from GitHub... ✔
--> Updating local ~/.dotfiles repository... ✔
--> Fetching daptiv/dotfiles from GitHub... ✔
--> Check current branch of daptiv/dotfiles against requested branch... ✔
--> Running dotfiles script/setup... ✔
Box strap is complete.
Next steps:
1. run 'strap-daptiv' to run daptiv-dotfiles scripts/setup
2. run 'strap-user' to run user dotfiles scripts/setup
```

### CODE REVIEW

File: bin/strap.sh [hunk 1]
```diff
@@ -3,6 +3,14 @@
 #/ Install development dependencies on macOS.
 set -e
 
+RED="\033[0;31m"
+YELLOW="\033[0;33m"
+GREEN="\033[0;32m"
+OCHRE="\033[38;5;95m"
+BLUE="\033[0;34m"
+WHITE="\033[0;37m"
+RESET="\033[0m"
```
Comment (jtrinklein) [inline bin/strap.sh]: 👏

File: bin/strap.sh [hunk 2]
```diff
@@ -279,8 +287,8 @@ fi
 logk
 
 # Install Homebrew Bundle, Cask and Services tap.
-log "Installing Homebrew taps and extensions:"
-brew bundle --file=- <<EOF
+logn "Installing Homebrew taps and extensions..."
+brew bundle --file=- 1>/dev/null 2>&1 <<EOF
```
Comment (jtrinklein) [inline bin/strap.sh]: we do want to be able to see this when we're in debug mode though....

File: bin/strap.sh [hunk 3]
```diff
@@ -325,35 +333,38 @@ if [ -n "$STRAP_GITHUB_USER" ]; then
   fi
 
   DOTFILES_URL="git@github.com:$DOTFILES_REPO"
-  log "Checking for remotes on $DOTFILES_URL"
+  logn "Checking for remotes on $DOTFILES_URL..."
   if git ls-remote "$DOTFILES_URL" &>/dev/null; then USER_DOTFILES_EXISTS=1; fi
+  logk
   if [ -n "$USER_DOTFILES_EXISTS" ]; then
-    log "Fetching $DOTFILES_REPO from GitHub:"
+    logn "Fetching $DOTFILES_REPO from GitHub..."
```
Comment (jtrinklein) [inline bin/strap.sh]: the problem here is that  if it doesn't exist... .you get 
```
Fetching git@github.com:jtrinklein/dotfiles from GitHub...Cloning to ~/.dotfiles...√√
```

File: bin/strap.sh (hunk 2 above)
Comment (jtrinklein) [reply bin/strap.sh]: i think.... maybe not though

Comment (heathsnow) [reply bin/strap.sh]: Update.

File: bin/strap.sh (hunk 3 above)
Comment (heathsnow) [reply bin/strap.sh]: Updated 'Cloning...' output to `logdebug`.

**Comment by heathsnow** [comment]
@jtrinklein Update with PR feedback.

**Pull request merged**